src/main/resources/team/asd/mapper/PartyMapper.xml

```diff
@@ -55,6 +55,15 @@
 		</if>
 	</select>
 
+	<select id="readByChannelMaskUserType" parameterType="java.lang.String" resultMap="PartyResultMap">
+		<bind name="channelNameStartsWith" value="mask + '%'"/>
+		SELECT
+		<include refid="partyColumnList"/>
+		FROM party JOIN channel_partner ON party.id = channel_partner.party_id
+		WHERE channel_name LIKE #{channelNameStartsWith}
+		AND user_type = #{userType}
```

Comment:
also add the table name for all fields, as it is not clear which table these fields refer to
`channel_name` and `user_type `

Reply:
@vasylnikorych done. Table aliases were added for all attributes in join scripts